app/forms/idv/profile_form.rb

```diff
@@ -35,11 +38,11 @@ def profile
     end
 
     def pii_attributes
-      @_pii_attributes ||= Pii::Attributes.new
+      @_pii_attributes ||= Pii::Attributes.new_from_hash(previous_address: OpenStruct.new)
```

Comment:
Why not use a regular class instead of an OpenStruct? The class is 1.5x faster.
```ruby
class PreviousAddress
  attr_accessor :address1, :address2, :city, :state, :zipcode
end
```
Instead of:
```ruby
delegate :prev_address1, :prev_address2, :prev_city, :prev_state, :prev_zipcode,
             to: :previous_address
```
it would be:
```ruby
attr_reader :prev_address1, :prev_address2, :prev_city, :prev_state, :prev_zipcode
```

and instead of:
```ruby
previous_address[new_key] = value
```
it would be
```ruby
previous_address.public_send("#{new_key}=", value)
```